gnomad_qc/v4/resources/basics.py

```diff
@@ -385,17 +385,14 @@ def get_logging_path(name: str, version: str = CURRENT_VERSION) -> str:
     return f"{qc_temp_prefix(version)}{name}.log"
 
 
-def add_meta(
-    mt: hl.MatrixTable, version: str = CURRENT_VERSION, meta_name: str = "meta"
-) -> hl.MatrixTable:
+def add_meta(mt: hl.MatrixTable, meta_name: str = "meta") -> hl.MatrixTable:
```

Comment:
it might still be helpful to be able to specify which meta version requested here, right?